specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCircuit.json

```diff
@@ -1260,6 +1442,78 @@
         }
       },
       "description": "Response for ListPeering API service call retrieves all peerings that belong to an ExpressRouteCircuit."
+    },
+	"ExpressRouteCircuitConnectionsPropertiesFormat": {
+      "properties": {
+	    "expressroutecircuitpeering": {
+			"$ref": "#/definitions/ExpressRouteCircuitPeering",
+			"description": "Reference to Express Route Circuit Private Peering Resource of the circuit initiating connection."
+		},
+		"peerexpressroutecircuitpeering": {
+			"$ref": "#/definitions/ExpressRouteCircuitPeering",
+			"description": "Reference to Express Route Circuit Private Peering Resource of the peered circuit."
+		},
+		"addressPrefix": {
+          "type": "string",
+          "description": "/29 IP address space to carve out Customer addresses for tunnels."
+        },
+        "authorizationKey": {
+          "type": "string",
+          "description": "The authorization key."
+        },
+        "circuitConnectionStatus": {
+          "type": "string",
+          "description": "Express Route Circuit Connection State. Possible values are: 'Connected' and 'Disconnected'.",
+          "enum": [
+            "Connected",
+            "Disconnected"
+          ],
+		  "readOnly": true, 
+          "x-ms-enum": {
+            "name": "ConnectionStatus",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+	"ExpressRouteCircuitConnections": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ExpressRouteCircuitConnectionsPropertiesFormat"
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Express Route Circuit Connection in an ExpressRouteCircuitPeering resource."
+    },
+    "ExpressRouteCircuitConnectionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExpressRouteCircuitConnections"
+          },
+          "description": "The Express Route Circuit Connections in an ExpressRoute Circuit Private Peering."
+        },
+        "nextLink": {
```

Comment:
mark this as readonly

Reply:
removed that item since we are not supporting list operation on circuit connection as of yet